bootstrap_atk_money_type.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace atk4\data;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\Type;
+use Doctrine\DBAL\Types\Types;
+
+// TODO for types to DBAL migration, remove later, there should be no type "money"
+
+class bootstrap_atk_money_type extends Type
```

Comment:
all tests pass with this super simple replacement with `decimal` native DBAL Type

@DarkSide666 @georgehristov should we completely remove `money` (database) type now in favor of native `decimal`?